remoteappmanager/docker/container_manager.py

```diff
@@ -162,8 +163,21 @@ def _start_container(self, user_name, image_name, volumes):
         # volumes = {volume_source: {'bind': volume_target,
         #                            'mode': volume_mode}
         volumes = volumes if volumes else {}
+
+        # Filter away the volume sources that do not exist,
+        # otherwise Docker would create non-existing host directory
+        # See Docker PR #21666
+        filtered_volumes = {source: volumes[source]
+                            for source in volumes
+                            if os.path.exists(source)}
```

Comment:
For the source path being a file, it will be mounted as a file.  It is a feature of docker that might be handy at times.  I don't see why we should stop that from happening.  Thought?